src/index.spec.js

```diff
@@ -161,11 +161,18 @@ describe('provideState', () => {
   describe('computed', () => {
     const sum = jest.fn(({ foo }, { bar }) => foo + bar)
     const circularComputed = jest.fn(({ circularComputed }) => {})
+    const throwComputed = jest.fn((_, { baz }) => {
```

Comment:
Unsupported: `baz` is not declared in `initialState`.

Reply:
I know, computed can be derived from props too. here `baz` is a prop.. Should we test only computed derived from State ?

Reply:
My bad, this is a `prop`, this is correct.